kratos/processes/levelset_convection_process.h

```diff
@@ -118,14 +118,13 @@ class LevelSetConvectionProcess
         ModelPart& rBaseModelPart,
         typename TLinearSolver::Pointer pLinearSolver,
         Parameters ThisParameters)
-        : LevelSetConvectionProcess(
-            rBaseModelPart,
-            ThisParameters)
```

Comment:
Why removing this call to the main protected constructor? Indeed, if you keep it you don't need to assign the member variables below (only to add the extra `mLevelSetConvectionSettings` one).